packages/core/test/test_util/expectAtLeastOneMatching.js

```diff
@@ -15,13 +15,16 @@ const { findAllMatchingItems, reportFailure } = require('./matchingItems');
  * @type {Function}
  * @param {Array.<InstanaBaseSpan>} items
  * @param {(span: InstanaBaseSpan) => void} expectations
- * @returns {InstanaBaseSpan}
+ * @param {*} options
+ * @returns {*}
  */
-module.exports = exports = function expectAtLeastOneMatching(items, expectations) {
+module.exports = exports = function expectAtLeastOneMatching(items, expectations, options = { numberOfMatches: null }) {
```

Comment:
For db2 I am calling `execute` twice for a prepared statement.
As we do not capture statement arguments, the span looks exactly the same.

See case "must trace prepare execute async with reusage"

Reply:
_question:_ Can we instead use `packages/core/test/test_util/expectExactlyNMatching` for this? Maybe I'm missing something here but this use case sounds like it is already covered by `expectExactlyNMatching`.

Reply:
Done 👍